app/src/main/java/io/stanwood/mhwdb/datasource/net/mhw/MhwArmor.kt

```diff
@@ -30,6 +29,6 @@ data class MhwArmor(
     val type: String,
     val rank: String,
     val name: String,
-    @Optional val assets: MhwArmorAsset? = null,
```

Comment:
Aren't optionals supported anymore, or don't we need them anymore?

Reply:
optional is deprecated

Reply:
Sweet! So now we can just have nullable types and thus tell the serializer they are optional? Very good! 😍